i18n/es/docusaurus-plugin-content-docs/current/build/subnet/upgrade/customize-a-subnet.md

```diff
@@ -1134,3 +1134,47 @@ El archivo `upgrades.json` que se muestra a continuación describe una actualiza
+
+```json
+{
+   "networkUpgradeOverrides": {
+     "{networkUpgrade1}": marcadetiempo1,
+     "{networkUpgrade2}": marcadetiempo2,
+   }
+}
+```
+
+La "marca de tiempo" debe ser una marca de tiempo de Unix en segundos.
+
+Por ejemplo, si se perdió la activación de Durango en Fuji (13 de febrero de 2024, 16:00 UTC) o Mainnet (6 de marzo de 2024, 16:00 UTC) y tiene problemas en su red, puede reprogramar la activación de Durango mediante actualizaciones. Para hacer esto, necesita preparar un nuevo update.json que incluya lo siguiente:
+
+```json
+{
+  "networkUpgradeOverrides": {
+    "durangoMarca de tiempo": 1712419200
+  }
+}
+```
+
+Esto reprograma la activación de Durango para el 6 de noviembre de 2024 a las 16:00:00 UTC (un mes después de la activación real). Después de preparar el Upgrade.json, debe actualizar el directorio de la cadena con el nuevo Upgrade.json y reiniciar los nodos. Debería ver registros similares al siguiente:
+
+```go
+INFO [03-22|14:04:48.284] <fPypUHjNvJqBKXBx2LEoJ9u5b8rRxMtEhb4v2QEDQejEiTtMG Chain> github.com/ava-labs/subnet-evm/plugin/evm/vm.go:367: Applying network upgrade overrides overrides="{\"durangoTimestamp\":1712419200}"
+...
+INFO [03-22|14:04:48.288] <fPypUHjNvJqBKXBx2LEoJ9u5b8rRxMtEhb4v2QEDQejEiTtMG Chain> github.com/ava-labs/subnet-evm/core/blockchain.go:335: Avalanche Upgrades (timestamp based):
+INFO [03-22|14:04:48.288] <fPypUHjNvJqBKXBx2LEoJ9u5b8rRxMtEhb4v2QEDQejEiTtMG Chain> github.com/ava-labs/subnet-evm/core/blockchain.go:335:  - SubnetEVM Timestamp:           @0          (https://github.com/ava-labs/avalanchego/releases/tag/v1.10.0)
+INFO [03-22|14:04:48.288] <fPypUHjNvJqBKXBx2LEoJ9u5b8rRxMtEhb4v2QEDQejEiTtMG Chain> github.com/ava-labs/subnet-evm/core/blockchain.go:335:  - Durango Timestamp:            @1712419200 (https://github.com/ava-labs/avalanchego/releases/tag/v1.11.0)
+...
+```
+
+Esto significa que su nodo está bloqueado y cargado para la nueva activación de Durango. Una vez alcanzada la nueva marca de tiempo, su nodo activará Durango y comenzará a procesar transacciones con las nuevas funciones de Durango.
+
+:::precaución
```

Comment:
```suggestion
:::caution
```

@dubrado changes [like this](https://add-network-reschedule.avalanche-docs.pages.dev/es/build/subnet/upgrade/customize-a-subnet#actualizaciones-de-red-reprogramaci%C3%B3n-de-actualizaciones-de-red-obligatorias) break the front end:

![image](https://github.com/ava-labs/avalanche-docs/assets/99047250/b5afdd83-babb-4447-8bf8-604ffe95b5b4)